GUIDE.md

```diff
@@ -0,0 +1,81 @@
+Pegaswitch Getting Started Guide
+================================
+A guide to getting your Switch from factory to running Pegaswitch.
+
+Requirements
+------------
+- A Nintendo Switch running FW 1.0.0 - 3.1.0
+	- For 1.0.0, you will also need a copy of [Puyo Puyo Tetris JP](https://www.amazon.com/gp/product/B01N4PKS4S/ref=oh_aui_detailpage_o00_s00?ie=UTF8&psc=1)
+- A PC running any OS capable of running Node.JS that is connected to a network your Switch can connect to
```

Comment:
This is hard to read. Try "A PC that can run Node.js." I don't think we need to mention the thing about the network.